app/controllers/teacher_interface/pages_controller.rb

```diff
@@ -3,10 +3,11 @@ class PagesController < BaseController
     before_action :load_eligibility_check
 
     def eligible
-      session[:eligibility_check_complete] = true
+      @eligibility_check.touch(:completed_at)
```

Comment:
Could we move this in to a `before_action` for both methods?

Reply:
I also wonder what happens if someone goes through the flow again, do we need to create a new `EligibilityCheck` if the current one is marked as complete or do we just take the user straight to the results page?

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/04863ce9a551c53d71c96325ef69f0af652e2cea/app/controllers/teacher_interface/base_controller.rb#L8